clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -3074,6 +3077,43 @@ private static void dumpTopicPartitionIngestionContext(CommandLine cmd) throws E
     }
   }
 
+  static void dumpHostHeartbeatLag(
+      TransportClient transportClient,
+      String topicFilter,
+      String partitionFilter,
+      String lagFilter) throws Exception {
+    String topicName = topicFilter == null ? "" : topicFilter;
```

Comment:
How about we just use null to replace "" to indicate we did not explicitly provide string topic and partition int partition number. For example,` ./admin-tool --server-url xxxx  --kafka-topic topic_2 --lag-filter-enabled false` to show all partitions' lag on that host.

Reply:
But it needs to be passed into the request URI. Will null be converted into "null"? or actually null will throw exception in this case?